.neuro/live-pachyderm.yml

```diff
@@ -285,49 +241,67 @@ jobs:
-      MLFLOW_STORAGE: ${{ params.mlflow_storage }}
-      MLFLOW_URI: ${{ params.mlflow_uri }}
       NEURO_PASSED_CONFIG: secret:platform-config
-      TRAIN_IMAGE_REF: ${{ images.train.ref }}
-      PACHY_PIPELINE_NAME: ${{ params.pachy_pipeline_name }}
     bash: |
-      git clone -b ${PROJECT_GIT_BRANCH} ${PROJECT_GIT_REPO} ${PROJECT}
-
       # Init pachctl
       pachctl config update context default --pachd-address ${PACHY_URI}
       pachctl config set active-context default
+      pachctl inspect repo "${PACHY_REPO}" || \
+          pachctl create repo "${PACHY_REPO}"
 
-      # Store Neuro-flow batch and project definition base-64 encoded
-      BATCH_B64=$(cat ${PROJECT}/.neuro/training.yaml | base64 -w 0)
-      PROJECT_B64=$(cat ${PROJECT}/.neuro/project.yml | base64 -w 0)
-
-      # Substitue Neuro-flow batch parameters from job ENV vars and store them b64 encoded
-      PARAMS_B64=$(cat ${PROJECT}/.neuro/training-batch-params.yaml | envsubst | base64 -w 0)
+      # Store neuro-flow project definition in base64
+      LIVE_B64=$(cat ${{ volumes.project.mount }}/.neuro/live.yml | base64 -w 0)
+      PROJECT_B64=$(cat ${{ volumes.project.mount }}/.neuro/project.yml | base64 -w 0)
 
       # Create a copy of pipeline template
-      cp $${PROJECT}/config/pipeline.json /tmp/pipeline.json
+      cp ${{ volumes.project.mount }}/config/pipeline.json /tmp/pipeline.json
+
+      # Set pipeline config parameters
       sed -i -e s/##NEURO_PASSED_CONFIG##/${NEURO_PASSED_CONFIG}/ /tmp/pipeline.json
+      sed -i -e s/##PACHY_REPO##/${PACHY_REPO}/ /tmp/pipeline.json
 
-      # Probagate project and batch definition and its parameters into the Pachyderm pypeline definition
-      sed -i -e s/##PROJECT_B64##/${PROJECT_B64}/ -e s/##BATCH_B64##/${BATCH_B64}/ -e s/##PARAMS_B64##/${PARAMS_B64}/ /tmp/pipeline.json
+      # Propagate project definition to the Pachyderm pipeline
+      sed -i -e s/##PROJECT_B64##/${PROJECT_B64}/ -e s/##LIVE_B64##/${LIVE_B64}/ /tmp/pipeline.json
 
       # Set the pipeline name
-      sed -i -e s/##PIPELINE_NAME##/${PACHY_PIPELINE_NAME}/ /tmp/pipeline.json
+      sed -i -e s/##PIPELINE_NAME##/${{ params.pachy_pipeline_name }}/ /tmp/pipeline.json
 
       # Create pipeline
-      pachctl list pipeline $PACHY_PIPELINE_NAME >/dev/null && pachctl delete pipeline $PACHY_PIPELINE_NAME >/dev/null
+      pachctl list pipeline ${{ params.pachy_pipeline_name }} >/dev/null && \
+          pachctl delete pipeline ${{ params.pachy_pipeline_name }} >/dev/null
       pachctl create pipeline -f /tmp/pipeline.json
-      echo "Pipeline '${PACHY_PIPELINE_NAME}' created successfully."
+
+      echo "Pipeline '${{ params.pachy_pipeline_name }}' created successfully."
+
+  train:
+    image: $[[ images.train.ref ]]
+    preset: cpu-medium
+    life_span: 1d
+    params:
+      mlflow_storage:
+        descr: Storage path, where MLFlow server stores model binaries
+        default: 'storage:/${{ project.owner }}/${{ flow.id }}/mlruns'
```

Comment:
`${{ flow.id }}` is deprecated, let's use `${{ flow.project_id }}`, instead https://neu-ro.gitbook.io/neuro-flow/reference/project-configuration-syntax#id or `${{ flow.flow_id }}`